app/sidekiq/evss/disability_compensation_form/form0781_document_upload_failure_email.rb

```diff
@@ -109,20 +116,20 @@ def send_notification_mailer(submission)
         log_mailer_dispatch(submission, log_info, notify_response)
       end
 
-      def log_mailer_dispatch(submission, log_info, email_response = {})
+      def log_mailer_dispatch(_submission, log_info, _email_response = {})
         StatsD.increment("#{STATSD_METRIC_PREFIX}.success")
 
         Rails.logger.info('Form0781DocumentUploadFailureEmail notification dispatched', log_info)
 
-        cl = caller_locations.first
-        call_location = Logging::CallLocation.new(ZSF_DD_TAG_FUNCTION, cl.path, cl.lineno)
-        zsf_monitor = ZeroSilentFailures::Monitor.new(Form526Submission::ZSF_DD_TAG_SERVICE)
+        # cl = caller_locations.first
+        # call_location = Logging::CallLocation.new(ZSF_DD_TAG_FUNCTION, cl.path, cl.lineno)
+        # zsf_monitor = ZeroSilentFailures::Monitor.new(Form526Submission::ZSF_DD_TAG_SERVICE)
```

Comment:
Did you mean to leave this code commented out for some reason? Assume you were planning on removing this